web/services/password_resetter.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Looseleaf.PasswordResetter do
+  alias Looseleaf.{Email, Mailer, User}
+
+  def reset_password(email, repo) do
```

Comment:
Any reason you're passing the repo in? are you doing that for all services like this?